core/src/main/scala/zio/config/Config.scala

```diff
@@ -4,56 +4,52 @@ import java.io.{ File, FileInputStream }
 import java.util.Properties
 
 import zio.system.System
-import zio.{ system, IO, Tagged, ZEnv, ZIO, ZLayer }
+import zio.{ Layer, Tagged, ZIO, ZLayer }
```

Comment:
For these methods, I'd make sure that:

1. For every constructor `ConfigSource` (`fromXYZ`), we have a corresponding constructor here, and with the same name.
2. That all methods in this object are alphabetized, with operators appearing first; and private methods moved to a separate section at the end of the object.